fix(connection indicator): rework as list

joshuai96 opened the pull request:
When using tables  it's should be structured with an according header section.

This is related to [WCAG Success Criterion 1.3.1 Info and Relationships](https://www.w3.org/TR/WCAG22/#info-and-relationships).

Addes some missing german translations after I added my table header translations.

joshuai96 committed:
fix(connection-idicator): rework table as list

Signed-off-by: Joshua Irmer <irmer@gonicus.de>

joshuai96 committed:
lang: add missing german translations

Signed-off-by: Joshua Irmer <irmer@gonicus.de>

jitsi-jenkins commented:
Hi, thanks for your contribution!
If you haven't already done so, could you please make sure you sign our CLA (https://jitsi.org/icla for individuals and https://jitsi.org/ccla for corporations)? We would unfortunately be unable to merge your patch unless we have that piece :(.

saghul commented:
We don't want a visible table header there. Is there a way around it? Else we'll have to shrug it off and not be compliant on that component. The fact that it's rendered a sa table is an implementation detail, it could just be a list of items...

joshuai96 commented:
I'm quite glad  to hear that. I didn't like the header either. 
We are currently working of fixing in the most easiest way to prepare for the next WCAG compliance test at the start of Q2.

I would prefer rework as a list. I'm going to propose a change to this and come back with it here :)

saghul commented:
Sounds good!

joshuai96 commented:
Reworked the table as a flex list.

Sadly the same size fields are lost because of this, as I didn't want to provide a given width for the fields, as text length might change with different languages.

Even though I removed my own translations, I kept the missing German translations.

saghul commented:
> Sadly the same size fields are lost because of this, as I didn't want to provide a given width for the fields, as text length might change with different languages.

We'd want to keep the save UX as before though, for this to be acceptable.

joshuai96 commented:
![screenshot](https://github.com/user-attachments/assets/68308ad4-72ea-4d9f-a180-ccb6f71fa047)

I've opted to go the route of a fixed list width and 50% labels. Allowing the text to wrap should accommodate long lines. For a table that isn't as nice but for a list it's fine.

joshuai96 commented:
@saghul 
As discussed I'll mark you on the PRs that might be re-opened.

This one might have some discussion left to do
